src/index.js

```diff
@@ -1,11 +1,36 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { Provider } from 'react-redux';
+import App from './components/App';
+import store from './reducers/index';
+import { createBook } from './actions/index';
+
+const initialState = [
+  {
+    bookId: Math.floor(Math.random() * 1000),
+    title: 'Siddartha',
+    category: 'Biography',
+  },
+  {
+    bookId: Math.floor(Math.random() * 1000),
+    title: 'Doors of perception',
+    category: 'Learning',
+  },
+  {
+    bookId: Math.floor(Math.random() * 1000),
+    title: 'Divine Comedy',
+    category: 'Comedy',
+  },
+];
+
+initialState.map((book) => store.dispatch(createBook(book)));
```

Comment:
- [ ] [REQUIRED] Good job by trying to pass the initial state to Book. However, the proper way to do it is like this:

```JS
import { createStore } from 'redux';
import books from './reducers/books';
const store = createStore(books, initialState);
```

Feel free to check this link : [Initializing-state](https://redux.js.org/usage/structuring-reducers/initializing-state)

Reply:
Issue fixed